src/gsi/control2state.f90

```diff
@@ -1,3 +1,58 @@
+!-------------------------------------------------------------------------
+!    NOAA/NCEP, National Centers for Environmental Prediction GSI        !
+!-------------------------------------------------------------------------
+!BOP
+!
+! !MODULE:  control2state_mod --- control2state_mod variables and routines
+!
+! !INTERFACE:
+!
+module control2state_mod
+
+! !USES:
+
+
+! !DESCRIPTION: module control2state routines and variables
+
+use kinds, only: r_kind,i_kind
+use control_vectors, only: control_vector,c2sset_flg
+use gsi_4dvar, only: nsubwin, l4dvar, lsqrtb,ladtest_obs
+use gsi_chemguess_mod, only: gsi_chemguess_get
+use gsi_metguess_mod, only: gsi_metguess_get
+use gsi_bundlemod, only: gsi_bundlegetpointer
+use gsi_bundlemod, only: gsi_bundlecreate
+use gsi_bundlemod, only: gsi_bundle
+use gsi_bundlemod, only: gsi_bundlegetvar
+use gsi_bundlemod, only: gsi_bundleputvar
+use gsi_bundlemod, only: gsi_bundledestroy
+use gsi_bundlemod, only: assignment(=)
+use gridmod, only: nems_nmmb_regional
+use gridmod, only: regional, twodvar_regional            
+
+implicit none
+
+private
+public :: do_getprs,do_normal_rh_to_q,do_tv_to_tsen,do_getuv,do_cw_to_hydro
+public :: do_cw_to_hydro_hwrf,nclouds,ngases
+public :: control2state
+public :: control2state_ad
```

Comment:
@jderber-NOAA 
I don't see the removal of the control2state_ad.f90 file, though I'm fairly sure you intended to remove it, correct?

Reply:
Yes.  Removed.  Thanks.